napari_aicssegmentation/assets/icons/gear.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?><svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg"
```

Comment:
This needs to be added to the `MANIFEST.in` file so that it gets copied over into the package. There are already examples in the file for other assets. I think it's safe to just do a recursive-include of all .svg files under ` napari_aicssegmentation/assets/icons`

Reply:
The current `MANIFEST.in` includes this line: `recursive-include napari_aicssegmentation/assets/icons *`
That should be sufficient, yeah?

Reply:
Yep that is fine. Didn't realize it was added before, should've checked.